crates/sargon/src/profile/mfa/mfa_factor_sources/off_device_mnemonic_factor_source/off_device_mnemonic_factor_source.rs

```diff
@@ -112,6 +112,10 @@ impl BaseBaseIsFactorSource for OffDeviceMnemonicFactorSource {
     fn name(&self) -> String {
         self.hint.label.value.clone()
     }
+
+    fn set_name(&mut self, updated: String) {
```

Comment:
minor improvement would be to change `String` -> `impl AsRef<str>` making it a bit easier to write unit tests, allowing you to pass string literals (&str) or `String` :)

Reply:
and inside this function you do `updated.as_ref().to_owned()`